src/pages/workspace/content/create/index.tsx

```diff
@@ -1,12 +1,35 @@
 import { serverSideTranslations } from "next-i18next/serverSideTranslations";
 import React from "react";
 import dynamic from "next/dynamic";
-
+import Header from "@/components/layouts/header/Header";
+import { useTheme } from "@mui/material/styles";
+import { Box, useMediaQuery } from "@mui/material";
 // @ts-ignore
 const Create = dynamic(() => import("editor/Create"), { ssr: false });
 
 const create = () => {
-  return <Create />;
+  const [isSidebarOpen, setSidebarOpen] = React.useState(true);
+  const [isMobileSidebarOpen, setMobileSidebarOpen] = React.useState(false);
```

Comment:
@suvarnakale code duplicate code on each file... find out some common place to keep and reuse

Reply:
ok